src/components/Restoration/CorrectNameRequest.vue

```diff
@@ -177,71 +167,61 @@ export default class CorrectNameRequest extends Vue {
             'OK'
           )
         } else {
-          this.parseNameRequest(nr)
-          this.emitIsSaved(true)
+          // set new data
+          this.emitNameRequest(nr)
+          this.emitApprovedName(this.getNrApprovedName(nr))
+          this.emitSaved(true)
         }
-      } catch {
-        // "validateNameRequest" handles its own errors
-        // Inform parent process is complete
-        this.emitIsSaved()
+      } catch (error) {
+        alert(error.message)
+        // inform parent that process is complete
+        this.$refs.form.resetValidation()
+        this.emitSaved(false)
       }
     }
   }
 
-  /**
-   * Parse and Set the Name Request date to Store.
-   * @param nr The name request data
-   */
-  private parseNameRequest (nr: NrResponseIF): void {
-    const nrCorrection: NrCorrectionIF = {
```

Comment:
This was some weird thing to convert from a proper NR object to some intermediate (local only) object, and then in filing-template-mixin only a few fields from this object are saved.

Reply:
This component is now in BCRS shared repo.